miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java

```diff
@@ -178,8 +180,12 @@ protected PoolDto asDto(Pool model) {
   public PoolDto createPool(@RequestBody PoolDto pool, UriComponentsBuilder uriBuilder, HttpServletResponse response)
       throws IOException {
     Pool poolobj = Dtos.to(pool);
-    if (poolobj.getVolume() == null && poolobj.getPoolableElementViews().stream().allMatch(view -> view.getDilutionVolumeUsed() != null)) {
-      poolobj.setVolume(poolobj.getPoolableElementViews().stream().mapToDouble(view -> view.getDilutionVolumeUsed()).sum());
+    if (poolobj.getVolume() == null && poolobj.getPoolDilutions().stream()
+        .map(PoolDilution::getPoolableElementView)
+        .allMatch(view -> view.getDilutionVolumeUsed() != null)) {
+      poolobj.setVolume(poolobj.getPoolDilutions().stream()
+          .map(PoolDilution::getPoolableElementView)
+          .mapToDouble(view -> view.getDilutionVolumeUsed()).sum());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)